template.xml

```diff
@@ -147,6 +108,26 @@
         </list>
        </t>
    </section>
+
+  <section title="Contributors">
+  <t>The GeoJSON draft is primarily the work of the following individuals.</t>
+  
+  <t>Howard Butler<vspace/>Hobu Inc.<vspace blankLines="1"/>Email: howard@hobu.co<vspace blankLines="1"/></t>
+  <t>Martin Daly<vspace/>Cadcorp<vspace blankLines="1"/>Email: martin.daly@cadcorp.com<vspace blankLines="1"/></t>
+  <t>Allan Doyle<vspace/>MIT<vspace blankLines="1"/>Email: adoyle@intl-interfaces.com<vspace blankLines="1"/></t>
+  <t>Tim Schaub<vspace/>Planet Labs<vspace blankLines="1"/>Email: tim.schaub@gmail.com<vspace blankLines="1"/></t>
+  <t>Stefan Hagen<vspace/>Rheinaustr. 62<vspace/>Bonn 53225<vspace/>DE<vspace/><vspace blankLines="1"/>Email: stefan@hagen.link<vspace/>URI: http://stefan-hagen.website/<vspace blankLines="1"/></t>
+
+  </section>
+  <section title="Acknowledgements">
```

Comment:
I disagree. This might be an appropriate place to acknowledge the original draft to meet the terms of the CC-BY license.

I prefer to retain some history.